manifests/init.pp

```diff
@@ -69,6 +69,20 @@
 #
 # $candlepin_manage_db:: Boolean indicating whether a database should be installed, this includes db creation and user
 #
+# $manage_candlepin::   Whether to manage candlepin
+#
+# $manage_qpid::        Whether to manage qpid
+#
+# $manage_pulp::        Whether to manage pulp. Currently requires $manage_application and $manage_candlepin to be true as well.
+#
+# $manage_application:: Whether to manage application (katello web ui)
```

Comment:
I don't really like the name of this, how about `manage_katello` as that fits with the other params naming convention like `manage_candlepin` ect.. ?

Reply:
Naming is hard and I'm fully open to other names. The reason I chose application was that this module is also named katello so ```$katello::manage_katello``` felt odd. UI can't be used because it also includes other aspects of it (tasks/dynflow daemon). That said, I'm not attached to ```manage_application```.

Reply:
Yea, agreed i dont like `$katello::manage_katello`. I'll think on it.